healthcheck/tools/db/config.go

```diff
@@ -77,10 +79,18 @@ func NewConfig(app *kingpin.Application, envUser string, envPassword string) *Co
 		"username",
 		"mongodb auth username, this flag or env var "+envUser+" is required",
 	).Envar(envUser).Required().StringVar(&db.DialInfo.Username)
-	app.Flag(
-		"password",
-		"mongodb auth password, this flag or env var "+envPassword+" is required",
-	).Envar(envPassword).Required().StringVar(&db.DialInfo.Password)
+	pass, err := ioutil.ReadFile("/etc/users-secret/MONGODB_CLUSTER_MONITOR_PASSWORD")
+	if err != nil && err != os.ErrNotExist {
+		return nil, fmt.Errorf("read MONGODB_CLUSTER_MONITOR_PASSWORD: %v", err)
```

Comment:
let's user `errors` package instead of fmt

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/551/commits/901f5a3d5126c2724aa55c310b3bf0a791c18ecc